theme/js/components/search/search.vue

```diff
@@ -1,184 +1,175 @@
+    <div class="fr-grid-row fr-grid-row--middle justify-between search-bar" ref="searchRef">
+      <SearchInput
+        :onChange="handleSearchChange"
+        :value="queryString"
+        :placeholder="$t('Search for data...')"
+      />
+    </div>
+    <div class="fr-grid-row fr-mt-2w fr-pt-3v fr-mb-3v justify-between fr-grid-row--middle">
+      <p class="fr-text--lg fr-text--bold fr-my-0" v-if="totalResults">
+        {{ $t("X Results", totalResults) }}
+      </p>
+      <div class="fr-grid-row align-items-center">
+        <label for="sort-search" class="fr-col-auto fr-text--sm fr-m-0 fr-mr-1w">
+            {{$t('Sort by:')}}
+        </label>
+        <div class="fr-col">
+            <select
+              id="sort-search"
+              class="fr-select"
+              name="sort"
+              v-model="searchSort"
+              @change="handleSortChange"
+            >
+              <option value="">
+                {{$t('Relevance')}}
+              </option>
+              <option v-for="{value, label} in sortOptions" :value='value'>
+                {{label}}
+              </option>
+            </select>
         </div>
       </div>
     </div>
-    <div class="fr-my-9v text-align-center fr-hidden-md">
-      <button
-        class="fr-btn fr-btn--secondary fr-btn--secondary-grey-400"
-        @click="extendedForm = !extendedForm"
-        >{{ $t("Submit") }}</button>
-    </div>
-  </section>
-  <section class="search-results fr-mt-1w fr-mt-md-3w" ref="results" v-bind="$attrs">
-    <transition mode="out-in">
-      <div v-if="loading">
-        <Loader />
+    <div class="fr-grid-row fr-mt-1w fr-mt-md-3w">
+      <div class="fr-col-12 fr-col-md-4 fr-col-lg-3">
+        <nav class="fr-sidemenu" :aria-label="$t('Filter results')">
+          <div class="fr-sidemenu__inner">
+            <button class="fr-sidemenu__btn fr-mt-1w" hidden aria-controls="fr-sidemenu-wrapper" aria-expanded="false">{{$t('Filter results')}}</button>
+            <div class="fr-collapse" id="fr-sidemenu-wrapper">
+              <div class="fr-sidemenu__title fr-mb-3v">{{$t('Filter results')}}</div>
+              <div class="fr-grid-row fr-grid-row--gutters">
+                <div class="fr-col-12">
+                  <MultiSelect
+                    :placeholder="$t('Organizations')"
+                    :searchPlaceholder="$t('Search an organization...')"
+                    listUrl="/organizations/?sort=-followers"
+                    suggestUrl="/organizations/suggest/"
+                    entityUrl="/organizations/"
+                    :values="facets.organization"
+                    :onChange="handleFacetChange('organization')"
+                  />
+                </div>
+                <div class="fr-col-12">
+                  <MultiSelect
+                    :placeholder="$t('Formats')"
+                    :searchPlaceholder="$t('Search a format...')"
+                    suggestUrl="/datasets/suggest/formats/"
```

Comment:
I think we could use a listUrl here instead, based on https://www.data.gouv.fr/api/1/datasets/extensions/.

Reply:
done